src/Cli/dotnet/Program.cs

```diff
@@ -194,14 +194,20 @@ internal static int ProcessArgs(string[] args, TimeSpan startupTime, ITelemetry
                         nologo: nologo,
                         skipWorkloadIntegrityCheck: skipWorkloadIntegrityCheck);
 
+                    var getStarOptionPassed = parseResult.CommandResult.Tokens.Any(t =>
+                        t.Value.Contains("getProperty", StringComparison.OrdinalIgnoreCase) ||
+                        t.Value.Contains("getItem", StringComparison.OrdinalIgnoreCase) ||
+                        t.Value.Contains("getTargetResult", StringComparison.OrdinalIgnoreCase));
```

Comment:
Instead of just checking that the string contains these values, could we check that it starts with `/` or `-` followed by the value?